sdgx/data_models/inspectors/numeric.py

```diff
@@ -14,69 +14,132 @@ class NumericInspector(Inspector):
 
     This class is a subclass of `Inspector` and is designed to provide methods for inspecting
     and analyzing numeric data. It includes methods for detecting int or float data type.
+
+    In August 2024, we introduced a new feature that will continue to judge the positivity or
```

Comment:
Perhaps we should indicate the PR and release version here, rather than the date?

Reply:
Good idea, I also have another branch in development,  I'll release after merging another PR. Due to various reasons, we haven't released a new version for a long time :(

Reply:
Nerver mind, thanks for your work!